### PR TITLE
Initial Interview Prep page styles

### PR DESCRIPTION
**Pull request by lilyndk**
## Description
Initial basic page styles with incorporation of accordion component from FAQs to make the page scroll not too long.

I think the in-page "jump-nav" should be a separate task post-launch.

## Change Type
- [ ] Bug Fix
- [X] New Feature
- [ ] Code Refactor
- [ ] Mentor Update
- [ ] Documentation
- [ ] Other

## Related Issue
https://github.com/WomenCodingCommunity/WomenCodingCommunity.github.io/issues/40

## Screenshots
AFTER:
Screenshot 2024-04-24 at 18 52 36

BEFORE:
## Screenshots
Screenshot 2024-04-24 at 18 54 06

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [X] I checked and followed the [contributor guide](CONTRIBUTING.md) 
- [X] I have tested my changes locally.
- [X] I have added a screenshot from the website after I tested it locally

### CODE REVIEW

File: program-interview-preparation.html [hunk 1]
```diff
@@ -9,18 +9,91 @@ <h1>Interview Preparation</h1>
...
+                            <h2 class="h4 m-0 pr-3">How it works</h2>
+                            <div class="position-absolute top-0 right-0 h-100 d-flex align-items-center">
+                                {% include icons/plus.svg %}
+                            </div>
+                        </div>
+                    </a>
+                    <div class="collapse" id="faq-1">
+                        <div class="card card-body border-0 p-0">
+                            <p>Participating in our Peer CV Clinic is easy and fun!</p> 
+                            <p>Simply join our <a href="https://womencodingcommunity.slack.com/archives/C0703P2RUTV" target="_blank">Slack channel #cv-clinic</a> and submit your CV (PDF or Word format), alongside a brief description of the role you’re applying for.</p>
```
Comment (lilyndk) [inline program-interview-preparation.html]: @dricazenck as mentioned in Slack, I've replaced "resume" with "CV"

Comment (dricazenck) [inline program-interview-preparation.html]:
```suggestion
                            <p>Simply join our <a href="https://womencodingcommunity.slack.com/archives/C0703P2RUTV" target="_blank">Slack channel #cv-clinic</a> and submit your CV (PDF or Google Spreadsheet), alongside a brief description of the role you’re applying for.</p>
```

File: _sass/custom/_faq.scss [hunk 2]
```diff
@@ -11,7 +9,7 @@
     }
   
      h2 {
-      color: $black;
+      color: $primary;
```
Comment (dricazenck) [inline _sass/custom/_faq.scss:12]: is this related to another change right? was that intended?

Comment (lilyndk) [reply _sass/custom/_faq.scss:12]: @dricazenck yes, I changed the faq/accordion link titles to use our purple link color for now, to make it more clear that you can click on it.

File: program-interview-preparation.html [hunk 3]
```diff
@@ -9,18 +9,91 @@ <h1>Interview Preparation</h1>
...
         <div class="section">
-            <h2><a href="#resume-clinic">Resume Clinic</a></h2>
-            <p>TODO</p>
+            <h2>CV Clinic</h2>
+            <p>Are you ready to take your CV to the next level? Join us for an exciting opportunity to elevate your professional profile with our CV Clinic! 🚀</p>
+            <p>At the Women Coding Community, we believe in the power of collaboration and support. That's why we're thrilled to offer our members a unique chance to receive personalized feedback and guidance on their CVs from their peers.</p> 
+            <p>Our CV Clinic is designed to provide you with valuable insights, expert tips, and constructive feedback to help you craft a standout CV that showcases your true potential.</p>
+            <div class="faq my-4">
+                <section id="accordion">
+                    <a class="py-3 d-block h-100 w-100 position-relative z-index-1 pr-1 text-secondary border-top" aria-controls="faq-1" aria-expanded="false" data-toggle="collapse" href="#faq-1" role="button">
```
Comment (dricazenck) [inline program-interview-preparation.html:18]: maybe not for now, but let's keep in mind to create a new class for this kind of things. The html is really big and hard to maintain. 
```suggestion
                    <a class="prep-link" aria-controls="interview-1" aria-expanded="false" data-toggle="collapse" href="#interview-1" role="button">
```

File: _sass/custom/_faq.scss (hunk 2 above)
Comment (lilyndk) [reply _sass/custom/_faq.scss:12]: Now that I've reused the accordion in body content on this page

File: program-interview-preparation.html [hunk 4]
```diff
@@ -9,18 +9,91 @@ <h1>Interview Preparation</h1>
...
+                        <div class="position-relative d-flex align-items-left">
+                            <h2 class="h4 m-0 pr-3">How it works</h2>
+                            <div class="position-absolute top-0 right-0 h-100 d-flex align-items-center">
+                                {% include icons/plus.svg %}
+                            </div>
+                        </div>
+                    </a>
+                    <div class="collapse" id="faq-1">
```
Comment (dricazenck) [inline program-interview-preparation.html]:
```suggestion
                    <div class="collapse" id="interview-1">
```

File: program-interview-preparation.html [hunk 5]
```diff
@@ -9,18 +9,91 @@ <h1>Interview Preparation</h1>
...
+            <h2>CV Clinic</h2>
+            <p>Are you ready to take your CV to the next level? Join us for an exciting opportunity to elevate your professional profile with our CV Clinic! 🚀</p>
+            <p>At the Women Coding Community, we believe in the power of collaboration and support. That's why we're thrilled to offer our members a unique chance to receive personalized feedback and guidance on their CVs from their peers.</p> 
+            <p>Our CV Clinic is designed to provide you with valuable insights, expert tips, and constructive feedback to help you craft a standout CV that showcases your true potential.</p>
+            <div class="faq my-4">
+                <section id="accordion">
+                    <a class="py-3 d-block h-100 w-100 position-relative z-index-1 pr-1 text-secondary border-top" aria-controls="faq-1" aria-expanded="false" data-toggle="collapse" href="#faq-1" role="button">
+                        <div class="position-relative d-flex align-items-left">
+                            <h2 class="h4 m-0 pr-3">How it works</h2>
+                            <div class="position-absolute top-0 right-0 h-100 d-flex align-items-center">
```
Comment (dricazenck) [inline program-interview-preparation.html:21]:
```suggestion
                            <div class="prep-icon">
```

File: program-interview-preparation.html [hunk 6]
```diff
@@ -9,18 +9,91 @@ <h1>Interview Preparation</h1>
...
+                        <div class="card card-body border-0 p-0">
+                            <p>Participating in our Peer CV Clinic is easy and fun!</p> 
+                            <p>Simply join our <a href="https://womencodingcommunity.slack.com/archives/C0703P2RUTV" target="_blank">Slack channel #cv-clinic</a> and submit your CV (PDF or Google Spreadsheet), alongside a brief description of the role you’re applying for.</p>
+                            <p>Our fellow members review each other's CVs, exchange feedback, and brainstorm ideas for improvement.</p>
+                            <p>Whether you're a seasoned professional or just starting out in your career, our CV Clinic is the perfect chance to refine your CV and take your CV to the next level.</p>
+                        </div>
+                    </div>
+                </section>
+                <section id="accordion">
+                    <a class="py-3 d-block h-100 w-100 position-relative z-index-1 pr-1 text-secondary border-top" aria-controls="faq-2" aria-expanded="false" data-toggle="collapse" href="#faq-2" role="button">
```
Comment (dricazenck) [inline program-interview-preparation.html]:
```suggestion
                    <a class="py-3 d-block h-100 w-100 position-relative z-index-1 pr-1 text-secondary border-top" aria-controls="interview-2" aria-expanded="false" data-toggle="collapse" href="#interview-2" role="button">
```

File: program-interview-preparation.html [hunk 7]
```diff
@@ -9,18 +9,91 @@ <h1>Interview Preparation</h1>
...
+                        <div class="position-relative d-flex align-items-left">
+                            <h2 class="h4 m-0 pr-3">Benefits of joining</h2>
+                            <div class="position-absolute top-0 right-0 h-100 d-flex align-items-center">
+                                {% include icons/plus.svg %}
+                            </div>
+                        </div>
+                    </a>
+                    <div class="collapse" id="faq-2">
```
Comment (dricazenck) [inline program-interview-preparation.html]:
```suggestion
                    <div class="collapse" id="interview-2">
```

File: program-interview-preparation.html (hunk 3 above)
Comment (lilyndk) [reply program-interview-preparation.html:18]: got it, yeah a lot of this is from the old code base. Definitely want to create a lot of reusable style patterns post-launch and reusable components such as this accordion

File: program-interview-preparation.html [hunk 8]
```diff
@@ -9,18 +9,91 @@ <h1>Interview Preparation</h1>
...
+                <section id="accordion">
+                    <a class="py-3 d-block h-100 w-100 position-relative z-index-1 pr-1 text-secondary border-top" aria-controls="faq-3" aria-expanded="false" data-toggle="collapse" href="#faq-3" role="button">
+                        <div class="position-relative d-flex align-items-left">
+                            <h2 class="h4 m-0 pr-3">Benefits of joining</h2>
+                            <div class="position-absolute top-0 right-0 h-100 d-flex align-items-center">
+                                {% include icons/plus.svg %}
+                            </div>
+                        </div>
+                    </a>
+                    <div class="collapse" id="faq-3">
```
Comment (dricazenck) [inline program-interview-preparation.html]:
```suggestion
                    <div class="collapse" id="interview-3">
```

File: program-interview-preparation.html [hunk 9]
```diff
@@ -9,18 +9,91 @@ <h1>Interview Preparation</h1>
...
+                                <li>Collaborative learning</li>
+                                <li>Accountability and motivation</li>
+                                <li>Gain confidence</li>
+                                <li>Different prespectives</li>
+                            </ul>
+                        </div>
+                    </div>
+                </section>
+            </div>
+            <p><a href="" target="_blank">Join us in our Leetcode Slack channel</a></p>
```
Comment (dricazenck) [inline program-interview-preparation.html]:
```suggestion
            <p><a href="https://womencodingcommunity.slack.com/archives/C0715E7E5T2" target="_blank">Join us in our Leetcode Slack channel</a></p>         
```

File: _sass/custom/_faq.scss (hunk 2 above)
Comment (lilyndk) [reply _sass/custom/_faq.scss:12]: We can also disregard this change for now until we have some accordion designs in Figma :) @dricazenck

File: program-interview-preparation.html [hunk 10]
```diff
@@ -9,18 +9,91 @@ <h1>Interview Preparation</h1>
...
+                                <li><u>New Strategies:</u> Sharpen your resume-writing skills and learn new techniques to make your resume stand out to potential employers.</li>
+                                <li><u>Confidence Boost:</u> Gain confidence in your professional profile and feel empowered to take the next step in your career journey.</li>
+                            </ul>
+                        </div>
+                    </div>
+                </section>
+            </div>
+            <p class="font-weight-bold">Ready to Elevate Your CV?</p>
+            <p>Don't miss out on this exciting opportunity to take your resume to new heights! 🌟</p>
+            <p><a href="https://womencodingcommunity.slack.com/archives/C0703P2RUTV" target="_blank">Join us in our #cv-clinic Slack channel</a></p>
```
Comment (dricazenck) [inline program-interview-preparation.html]:
```suggestion
            <p><a href="https://womencodingcommunity.slack.com/archives/C0703P2RUTV" target="_blank">Join us in our CV Clinic Slack channel</a></p>      
```

File: program-interview-preparation.html [hunk 11]
```diff
@@ -9,18 +9,91 @@ <h1>Interview Preparation</h1>
...
+                            <h2 class="h4 m-0 pr-3">How it works</h2>
+                            <div class="position-absolute top-0 right-0 h-100 d-flex align-items-center">
+                                {% include icons/plus.svg %}
+                            </div>
+                        </div>
+                    </a>
+                    <div class="collapse" id="faq-1">
+                        <div class="card card-body border-0 p-0">
+                            <p>Participating in our Peer CV Clinic is easy and fun!</p> 
+                            <p>Simply join our <a href="https://womencodingcommunity.slack.com/archives/C0703P2RUTV" target="_blank">Slack channel #cv-clinic</a> and submit your CV (PDF or Google Spreadsheet), alongside a brief description of the role you’re applying for.</p>
```
Comment (dricazenck) [inline program-interview-preparation.html]:
```suggestion
                            <p>Simply join our <a href="https://womencodingcommunity.slack.com/archives/C0703P2RUTV" target="_blank">Slack CV Slack channel</a> and submit your CV (PDF or Google Spreadsheet), alongside a brief description of the role you’re applying for.</p>
```

File: program-interview-preparation.html [hunk 12]
```diff
@@ -9,18 +9,91 @@ <h1>Interview Preparation</h1>
...
+            <p>At Women Coding Communities, we understand the importance of practice in mastering interviews. That's why we're excited to introduce our latest project: "Mock Interviews." This initiative is designed to provide you with the practice and support you need to excel in real interview scenarios.</p>
+            <p>Through mock interviews, you'll have the opportunity to refine your responses, develop effective strategies, and boost your confidence. Our goal is to help you feel more prepared and comfortable when facing the pressures of a real interview.</p>
+            <p>We suggest two distinct formats: behavioral and technical interviews. Whether you're looking to polish your soft skills or fine-tune your technical knowledge, we have tailored mock interviews to suit your needs.</p>
+            <p><a href="https://womencodingcommunity.slack.com/archives/C06VDMVGTDL" target="_blank">Join us on Slack in the #proj-mock-interviews</a> channel to indicate your interest and connect with others seeking practice. Pair up with fellow members and start honing your interview skills today!</p>
```
Comment (dricazenck) [inline program-interview-preparation.html]:
```suggestion
            <p><a href="https://womencodingcommunity.slack.com/archives/C06VDMVGTDL" target="_blank">Join us on Slack in the Mock interviews</a> channel to indicate your interest and connect with others seeking practice. Pair up with fellow members and start honing your interview skills today!</p>
```

**Review by dricazenck — APPROVED** [review]
